### PR TITLE
debug path command

### PR DESCRIPTION
**Pull request by qndel**
image

### CODE REVIEW

**Review by ephphatha — APPROVED** [review]
really useful command, thanks :D

File: Source/engine/render/scrollrt.cpp [hunk 1]
```diff
@@ -505,6 +505,12 @@ void DrawCell(const Surface &out, Point tilePosition, Point targetBufferPosition
 	const uint16_t levelPieceId = dPiece[tilePosition.x][tilePosition.y];
 	const MICROS *pMap = &DPieceMicros[levelPieceId];
 
+	uint8_t *tbl = nullptr;
```
Comment (glebm) [inline Source/engine/render/scrollrt.cpp:508]:
```suggestion
	const uint8_t *tbl = nullptr;
```

File: Source/engine/render/dun_render.hpp [hunk 2]
```diff
@@ -263,9 +263,10 @@ string_view MaskTypeToString(MaskType maskType);
  * @param levelCelBlock The MIN block of the level CEL file.
  * @param maskType The mask to use,
  * @param lightTableIndex The light level to use for rendering (index into LightTables).
+ * @param tbl Optional TRN for a tile.
```
Comment (glebm) [inline Source/engine/render/dun_render.hpp:266]: Perhaps we should remove the `lightTableIndex` and simply always pass a non-null `tbl`?

**Comment by AJenbo** [comment]
could you block it out with ifdef so it only affects debug builds

**Review by glebm — APPROVED** [review]

**Pull request merged**